src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -385,6 +389,7 @@ private static String suppressVariationsInInjectedValues(String content) {
     private static String replaceUnpredictableValuesWithPlaceholders(String content) {
         Date now = new Date();
         SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM yyyy, ");
+        sdf.setTimeZone(getTimeZone(content));
```

Comment:
May want to add comment about how this assumes that only one timezone would be used on the page.